### PR TITLE
Split README into multiple files

### PR DESCRIPTION
**Pull request by megahirt**
This PR splits the one long README into several files:

CONTRIBUTING.md
docs/ACKNOWLEDGEMENTS.md
docs/RELEASE.md
docs/DEVELOPER.md

It also moves the files in the former "docs" folder to a new folder called "sample_data", since that is what it actually is.  The "docs" folder is an open-source repo convention used for various markdown files like I have put above.  The CONTRIBUTING.md file is recognized by GitHub as a special file and will be displayed on the PR page.

A future PR will include more specific information on how to contribute.

### Type of Change

Documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have reviewed the title/description of this PR which will be used as the squashed PR commit message

### CODE REVIEW

**Review by longrunningprocess — APPROVED** [review]
I love this cleanup, thank you!  If you don't mind updating the PR description, I think this looks good to go in my mind.

File: README.md [hunk 1]
```diff
@@ -1,272 +1,27 @@
...
-### User Problems ###
+If you don't have a GitHub account or don't want your issue to be public, you can send an email to "languageforgeissues @ sil dot org".
 
-To report an issue using the **Language Forge** service, email "languageforgeissues @ sil dot org".
+## Other Resources ##
 
-## Special Thanks To ##
-- ![BrowserStack Logo](readme_images/browserstack-logo.png "BrowserStack") for mobile device testing.
-- [![Bugsnag logo](readme_images/bugsnag-logo.png "Bugsnag")](https://bugsnag.com/blog/bugsnag-loves-open-source) for error reporting.
+[Acknowledgements](docs/ACKNOWLEDGEMENTS.md)
```
Comment (longrunningprocess) [inline README.md:13]: These might look better on separate lines.

Comment (rmunn) [reply README.md:13]: I think an explicit `<br/>` at the end of each line might be needed. I believe Markdown uses trailing spaces before the newline character to indicate "Really break the line here", but that's a problem for most VS Code setups that trim trailing spaces.

Or, of course, turn this into an unordered list like:

```md
* Acknowledgements
* Known Issues
* etc.
```

**Comment by megahirt** [comment]
> I love this cleanup, thank you! If you don't mind updating the PR description, I think this looks good to go in my mind.

Yes, I pushed the PR quickly without taking the time to update the description.  Done, thank you.

File: README.md (hunk 1 above)
Comment (megahirt) [reply README.md]: I turned it into an unordered list

Comment (megahirt) [reply README.md]: I think @rmunn is fine with me resolving this.

**Pull request merged**